src/drive/mobile/modules/authorization/DriveMobileRouter.jsx

```diff
@@ -36,11 +35,7 @@ class DriveMobileRouter extends Component {
   async componentDidMount() {
     // Wait for the app to be booted to avoid race condition between cordova & JS
     await appBooted
-    const shouldMigrate = await shouldMigrateAdapter()
-    this.setState({
-      isAppBooted: true,
-      shouldDisplayMigrate: shouldMigrate
-    })
+    this.setState({ isAppBooted: true })
```

Comment:
nit: Maybe during this short PR, we could add a missing test on DriveMobileRouter ?

Reply:
All of the "Mobile" stuff will be removed pretty soon. Within ~ 1 or 3 months. Not sure this is the right time to add test to it.

Reply:
then I solve and approve

Reply:
On pourrait. J'ai peur que ce ne soit pas évident, et comme la release mobile de Drive est bloquée à cause de cette feature, je pense que l'idée ici est de débloquer la situation